src/main/java/it/gov/pagopa/rtd/ms/rtdmsfilereporter/feign/config/HttpClientConfig.java

```diff
@@ -37,6 +41,7 @@ CloseableHttpClient getHttpClient()
     PoolingHttpClientConnectionManager connectionManager = new PoolingHttpClientConnectionManager(
         registry);
 
-    return HttpClients.custom().setConnectionManager(connectionManager).build();
+    return apacheHttpClient5Configuration.createHttpClientBuilder().setConnectionManager(connectionManager)
```

Comment:
here you use the `httpClientBuilder` injected 
```suggestion
    return httpClientBuilder.setConnectionManager(connectionManager)
```